routes/api/user.js

```diff
@@ -3,46 +3,46 @@ const usersController = require('../../controllers/usersController');
 
 module.exports = (app) =>{
 
-    //sign up
-    app.post('/user',usersController.findOrCreate);
-        
+  //sign up
+  app.post('/user',usersController.findOrCreate);
 
+  app.post(
+    '/user/login',
+    (req, res, next) =>{
+      console.log('routes/user.js, login, req.body: ');
+      next()
+    },
+    passport.authenticate('local'),
+    (req, res) => {
+      console.log('logged in', req.user.email);
+      req.session.username = req.body.email;
+      
+      var userInfo = {
+        username: req.user.email
+      };
+      res.json(userInfo);
+  });
+  
+  app.get('/user', (req, res, next) => {
+    console.log('===== user!!======')
```

Comment:
do we need this? I feel like your consoles would be super cluttered when you are navigating around your project with so many console.logs everywhere.